src/views/clusteroverview/OverviewTab/vm-statuses-card/VMStatusItem.tsx

```diff
@@ -14,21 +14,24 @@ type VMStatusItemProps = {
   status: string;
   count: number;
   namespace?: string;
+  showIcon?: boolean;
 };
 
-const VMStatusItem: React.FC<VMStatusItemProps> = ({ status, count, namespace }) => {
+const VMStatusItem: React.FC<VMStatusItemProps> = ({
+  status,
+  count,
+  namespace,
+  showIcon = true,
```

Comment:
I think we can go with a different approach here to avoid adding the `showIcon` prop
`vmStatusIcon` on line 26 is an object to map a string value to an icon, but if we don't use some of the icons then maybe we should change the object to return null on the additional statuses so it won't show the icons and we avoid adding another prop

Reply:
Thank you Aviv! :)